scripts/pkg/build_templates/opensearch-dashboards/deb/Makefile

```diff
@@ -0,0 +1,16 @@
+TMP_PATH = /tmp/$(notdir $(CURDIR))
```

Comment:
This looks suspicious. Isn't there a TEMP env variable to reuse, hard-coding /tmp is likely not right.

Reply:
Replaced it by using the current directory.